src/apps/sources/components/SourceForm.tsx

```diff
@@ -290,10 +290,12 @@ const SourceForm: React.FC<Props> = ({
               </InputLabel>
               <Field
                 multiple
-                value={[]}
+                fullWidth
+                defaultValue={[]}
                 name="supported_locales"
                 id="supported_locales"
                 component={Select}
+                style={{ whiteSpace: "inherit !important" }}
```

Comment:
thank you @jwamalwa  cc @hadijahkyampeire

Reply:
Thanks @HerbertYiga for taking a look, I will also take a look